lcls_tools/superconducting/scLinac.py

```diff
@@ -802,20 +802,22 @@ def finish_characterization(self):
         print(f"{self} characterization successful")
     
     def walk_amp(self, des_amp, step_size):
-        print(f"walking {self} to {des_amp}")
+        print(f"walking {self} to {des_amp} from {self.selAmplitudeActPV.get()}")
         
-        while self.selAmplitudeDesPV.get() <= (des_amp - step_size):
+        while self.selAmplitudeActPV.get() <= (des_amp - step_size):
             self.check_abort()
             if self.quench_latch_pv.get() == 1:
                 raise utils.QuenchError(f"{self} quench detected, aborting rampup")
-            self.selAmplitudeDesPV.put(self.selAmplitudeDesPV.get() + step_size)
+            self.selAmplitudeDesPV.put(self.selAmplitudeActPV.get() + step_size)
             # to avoid tripping sensitive interlock
             sleep(0.1)
         
-        if self.selAmplitudeDesPV.get() != des_amp:
+        while not isclose(self.selAmplitudeActPV.get(), des_amp):
+            print(f"{self} not at desired ampltude, using caput")
```

Comment:
I'm not sure this print statement is easy to understand. How about:
```suggestion
            print(f"{self} not at desired amplitude, setting value again")
```